src/main/java/org/jenkinsci/account/Application.java

```diff
@@ -510,10 +510,12 @@ public AdminUI getAdmin() {
 
     public static final List<String> EMAIL_BLACKLIST = Arrays.asList(
         "@guerrillamail.com",
+        "@maildx.com",
         "@mailinator.com",
         "@mailnesia.com",
         "@sharklasers.com",
         "@thrma.com",
+        "@yandex.com",
```

Comment:
From my understanding yandex.com is like hotmail.com in Russia, this might not be a great idea :/